airbyte-integrations/connectors/destination-snowflake/build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
 airbyteJavaConnector {
     cdkVersionRequired = '0.11.0'
     features = ['db-destinations', 's3-destinations']
-    useLocalCdk = false
+    useLocalCdk = true
```

Comment:
temporary commit to test the changes with snowflake source, will revert this before merging

Reply:
Don't forget to bump the CDK version and publish in this PR as well!

Reply:
Yup! I plan to publish the cdk in this PR and will raise a separate PR to bump the version for snowflake destination